source/jobs/prerequisites/oracle.txt

```diff
@@ -136,7 +143,138 @@ Steps
                   :copyable: true
 
                   ALTER TABLE schemaName.tableName ADD SUPPLEMENTAL LOG DATA (ALL) COLUMNS;
-                  /* Additional ALTER TABLE STATEMENTS... */ 
+                  /* Additional ALTER TABLE STATEMENTS... */
+
+         .. step:: Create tablespaces
+
+            a. Create the required tablespaces:
+
+               .. code-block:: sql
+                  :copyable: true
+
+                  CONNECT sys/oraclepw@PDB_NAME as sysdba;
+                  CREATE TABLESPACE logminer_tbs DATAFILE '/opt/oracle/oradata/CDB_NAME/PDB_NAME/logminer_tbs.dbf'
+                  SIZE 25M REUSE AUTOEXTEND ON MAXSIZE UNLIMITED
+
+                  CONNECT sys/password@CDB_NAME as sysdba;
+                  CREATE TABLESPACE logminer_tbs DATAFILE '/opt/oracle/oradata/CDB_NAME/logminer_tbs.dbf'
+                  SIZE 25M REUSE AUTOEXTEND ON MAXSIZE UNLIMITED;
+
+               A continuous migration in a multi-tenant architecture requires 
+               a tablespace in each pluggable database and in the container 
+               database.
+
+            #. Assign quota for the tablespaces to the service user:
+
+               .. code-block:: sql
+                  :copyable: true
+
+                  ALTER USER <user> DEFAULT TABLESPACE LOGMINER_TBS QUOTA UNLIMITED ON LOGMINER_TBS CONTAINER=ALL;
+
+   .. tab:: Continuous Jobs (Amazon RDS)
+      :tabid: enable-continuous-jobs-rds
+
+      .. procedure::
+         :style: normal
+
+         .. step:: Set up user permissions
+
+            This procedure applies to Oracle instances which **are 
```

Comment:
np: i am not really sure why there is an emphasis on `**are hosted**` ? if we were to add an emphasis, it would make sense to put it on `**Amazon RDS**`, but given this entire section is already behind a tab, i think any more emphasis here is unnecessary